### DISCUSSION TITLE
Étude de cas : "Histoire du cinéma" P24

### DISCUSSION THREAD

**Discussion by jeromeBRM** [Hypertopic/HyperGlosae#212] [Show and tell]
Hello everyone, here is a video showcasing the features we added to Hyperglosae during the spring semester of 2024. This video was made in collaboration with @vernet-william, @EnzoSimonoviciUTT and @ToGoSleep.

1. Introduction - 00:00
2. New feature : Generate a timecode - 02:56
3. New feature : Translate to french - 04:55
4. Retrospective - 06:48

Please let us know in the discussion below if you have any feedback.

https://github.com/Hypertopic/HyperGlosae/assets/54836707/84c9909b-5cdc-46ce-ba86-00752d4971bc

**Comment by P-Laborderie** [comment]
Bonjour à toute l'équipe et mes félicitations pour ce beau travail ! Je tiens à vous dire que j'ai beaucoup apprécié votre capacité d'écoute lors de notre rencontre. Vous avez parfaitement identifié les besoins d'un chercheur en histoire du cinéma. Techniquement parlant, je ne peux pas juger en profondeur et dans le détail de la qualité de vos compétences en informatique. En revanche, je peux vous dire que les nouveautés que vous avez apportées sont très utiles et faciles à utiliser !
Très cordialement,

----------------------
Pascal LABORDERIE
Professeur des universités en sciences de l'information de de la communication
Coresponsable de l'équipe de recherche "Savoirs et apprentissages en formation, éducation, médiation et médiatisation" (SAFE2M-Cérep)
Université de Reims Champagne-Ardenne
+33 (0) 06 29 05 63 22
pascal.laborderie@univ-reims.fr
https://ircav.fr/member/pascal-laborderie/
